databricks/koalas/datetimes.py

```diff
@@ -188,7 +191,111 @@ def dayofyear(self) -> 'ks.Series':
         return _wrap_accessor_pandas(
             self, lambda s: s.dt.dayofyear, LongType()).alias(self._data.name)
 
+    @property
+    def quarter(self) -> 'ks.Series':
+        """
+        The quarter of the date.
+        """
+        return _wrap_accessor_pandas(
+            self, lambda s: s.dt.quarter, LongType()).alias(self._data.name)
+
+    @property
+    def is_month_start(self) -> 'ks.Series':
+        """
+        Indicates whether the date is the first day of the month.
```

Comment:
This seems having a doctest in Pandas side - https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.dt.is_month_start.html#pandas.Series.dt.is_month_start